### ISSUE TITLE
angular 15 issue import css

### ISSUE THREAD

**Issue by pookdeveloper** [openlayers/openlayers#14497]
**Describe the bug**
When import css in angular json it throws a error

Add ol.css to styles array in angular.json file:

  "styles": [
    "src/styles.scss",
    "node_modules/ol/ol.css"
  ],
 
 It dosent compile
 
![image](https://user-images.githubusercontent.com/3384734/217797759-d1a4a5b8-daa1-4539-8d88-5ee2de1d25f0.png)

**Comment by ahocevar** [comment]
Looks like something is misconfigured in your Angular setup. Do you have a repository or a stackblitz with a minimal application that shows the issue?

**Comment by pookdeveloper** [comment]
It fail when use in the .ts file , in angular.json like que image works well

**Comment by lonjoa** [comment]
I got the same problem when I upraded my angular version to 15.0.x.

**Comment by ahocevar** [comment]
Looks like a bug in Angular. The css shown in the error message is absolutely valid.

**Comment by jahow** [comment]
Isn't the `ol.css` file imported somewhere else in a typescript file? Importing CSS files from a TS file was apparently working in previous versions of Angular but not any more. And it looks like that is typically the reason for this error.

**Comment by SSlippa** [comment]
Having the same issue, any import in .ts file directly fails with the same error. We are using direct import in our microfrontends, for some purposes, no proper solution still :(

**Comment by lonjoa** [comment]
Hi, I solved my problem with removing 
import 'ol/ol.css' in one of the .ts files. 
Search for the import line in your code.

**Comment by SSlippa** [comment]
> Hi, I solved my problem with removing import 'ol/ol.css' in one of the .ts files. Search for the import line in your code.

Yep, this is a solution to eliminate an error, but we need this kind of import, we have a workaround, but this was a solution to avoid ng-deep
